controllers/svcdiscovery_controller_intg_test.go

```diff
@@ -0,0 +1,80 @@
+package controllers
+
+import (
+	goctx "context"
```

Comment:
Drop the alias since this is the only context import for the file.
```suggestion
	"context"
```